tools/enrollment_preprocessor.py

```diff
@@ -16,63 +20,89 @@ class User:
     first_name: str
     email: str
 
+    def full_name(self) -> str:
+        prefix = ""
+        if self.title:
+            prefix = f"{self.title} "
+        return f"{prefix}{self.first_name} {self.last_name}, {self.email}"
```

Comment:
Canonical representation on importers, as far as I know, is
> firstname lastname (email)

with email in parentheses. But no strong opinion.

Reply:
This would lead to ` Oma (Oma Abner (oma.abner@institution.example.com))` or so. The tool already separates the conflicting field value from the full name with parenthesis